### PR TITLE
Add boundaries endpoints

### PR DESCRIPTION
**Pull request by gabrieltron**

### CODE REVIEW

**Comment by gabrieltron** [comment]
[[Docs] Add boundaries documentation](https://trello.com/c/ma6zEftT/1692-docs-add-boundaries-documentation)

File: docs/field_boundaries_endpoints.md [hunk 1]
```diff
@@ -1261,6 +1588,71 @@ you can query for - and you can still query by the individual fields too.
 [Get intersection of fields][8] | <span class="badge badge--warning">POST</span> `/users/{id}/fields/intersect`
 [Delete a field][9] | <span class="badge badge--danger">DELETE</span> `/users/{id}/fields/{id}`
 
+### Boundary Resource
+
+Every Field at Leaf can have 0 or many boundaries. Only one boundary may be the active boundary, the others are inactive boundaries. Boundaries cannot be deleted or have its geometry updated. Every update generates a new Boundary, and Leaf keeps a history of all seen Boundaries.
```
Comment (CarlosSam) [inline docs/field_boundaries_endpoints.md:1593]: just as a reminder: Fields created through the Leaf's endpoints must have at least one boundary

**Review by CarlosSam — APPROVED** [review]
Nice! There is just a comment!

File: docs/field_boundaries_endpoints.md (hunk 1 above)
Comment (gabrieltron) [reply docs/field_boundaries_endpoints.md]: Good catch. Added in 8c8f018

**Pull request merged**